atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleWriteTable.java

```diff
@@ -28,6 +28,6 @@ public OracleWriteTable(
             ConnectionSupplier conns,
             OracleTableNameGetter oracleTableNameGetter,
             TableReference tableRef) {
-        super(config, conns, tableRef, new OraclePrefixedTableNames(config, conns, oracleTableNameGetter));
+        super(config, conns, tableRef, new OraclePrefixedTableNames(oracleTableNameGetter));
```

Comment:
seems like we could reduce the number of these classes we create and just have the OraclePrefixedTableNames passed in from the OracleDbTableFactory instead of the oracleTableNameGetter.